XNAControls/XNADropDown.cs

```diff
@@ -555,6 +573,9 @@ protected virtual void DrawItem(int index, int y)
 
             if (item.Text != null)
                 DrawStringWithShadow(item.Text, FontIndex, new Vector2(textX, y + 1), textColor);
+
+            if (item is XNADropDownDividerItem dividerItem)
+                DrawLine(new Vector2(0, y + dividerItem.LineY), new Vector2(Width, y + dividerItem.LineY), BorderColor);
```

Comment:
This is not really an object-oriented way to implement this.

A better way would be splitting item rendering to `XNADropDownItem`, so classes derived from that could draw themselves through overriding the rendering method. This `Draw` method would call that method for all of the dropdown items with fitting parameters (basically, the draw rectangle).